keras_nlp/layers/transformer_decoder.py

```diff
@@ -37,7 +37,7 @@ class TransformerDecoder(keras.layers.Layer):
     paper [Attention is All You Need](https://arxiv.org/abs/1706.03762). Users
     can instantiate multiple instances of this class to stack up a decoder.
 
-    This layer will always apply a causal mask to the decoder attention layer.
+    By default, this layer will apply a causal mask to the decoder attention layer.
```

Comment:
maybe add "This can be controlled by the `use_causal_mask` call argument.", so people know what to look for in our docs?

Reply:
`By default, this layer will apply a causal mask to the decoder attention layer.` from this I think `use_causal_mask` is already perfectly obvious and emphasizing it would just add unnecessary clutter to the documentation. 

Your call though, I am happy to add it if you feel like it would be helpful

Reply:
no this sounds good. let's keep what we got!